### PR TITLE
Changelogs

### PR DESCRIPTION
**Pull request by joseims**
Adding asperathos changelogs

### CODE REVIEW

**Review by matheuscmelo — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## Unreleased
+
+## Bootes - 16-08-2019
+### Added
+- New plugin architecture for the following Asperathos components:
```
Comment (liliarsampaio) [inline CHANGELOG.md]: When adding a new feature, as a user, I would expect a little explanation on what the feature is about, especially in this case when a "new plugin arch" sounds so generic. Just saying something along the lines: "New plugin architecture that allows for the addition of custom plugins based on specific user needs..". Not too much detail, but at least we know what the change is about.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+   - Monitor
+   - Manager
+   - Visualizer
+   - Controller
+- Persistent plugins for the following Asperathos components:
```
Comment (liliarsampaio) [inline CHANGELOG.md]: Rephrase to "Plugin persistence for the following components:"

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+   - Monitor
+   - Manager
+   - Visualizer
+   - Controller
+- PID scalling strategy at Asperathos Controller
```
Comment (liliarsampaio) [inline CHANGELOG.md]: Rephrase to "PID scaling strategy on the Controller component".

File: CHANGELOG.md (hunk 1 above)
Comment (liliarsampaio) [reply CHANGELOG.md]: Another thing is to remove "Asperathos" from the "Asperathos components" expression. Every change is related to Asperathos, no need to specify each time.

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+   - Visualizer
+   - Controller
+- Persistent plugins for the following Asperathos components:
+   - Monitor
+   - Manager
+   - Visualizer
+   - Controller
+- PID scalling strategy at Asperathos Controller 
+
+- New tests for kubejobs plugin at Asperathos Monitor
```
Comment (liliarsampaio) [inline CHANGELOG.md]: Highlight ```kubejobs```. Rephrase to "New tests for the kubejobs plugin of the Monitor component."

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+### Changed
+- Refactoring of the kubejobs plugin at Asperathos Manager
```
Comment (liliarsampaio) [inline CHANGELOG.md]: Highlight ```kubejobs```. Rephrase to "Refactoring of the kubejobs plugin of the Manager component."

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+###  Removed
+- Unused legacy plugins that were not working anymore
```
Comment (liliarsampaio) [inline CHANGELOG.md]: Never good to say something is "not working anymore" :) We should often say things are no longer supported, or not supported at the moment. That way, I would rephrase it to "... plugins that are not supported at the moment."

**Review by liliarsampaio — CHANGES_REQUESTED** [review]
some comments inline. thanks!

**Review by matheuscmelo — APPROVED** [review]
